aiidalab_widgets_base/utils/__init__.py

```diff
@@ -128,13 +129,13 @@ def _clear_value(self):
         else:
             self.value = ""
 
-    def show_temporary_message(self, value, clear_after=3):
+    def show_temporary_message(self, value):
```

Comment:
⚠️ This changes the API of this class!

If we really need the class variable, why not use this as a default here?

Reply:
Since I want to set this value when I create an instance of `StatusHTML`. Otherwise how I can set the timer to rather than 3 secs?

Reply:
> You can set the timer when you call that function.

I think the problem is, that `show_temporary_message` is called in the callback function. 

```
    @traitlets.observe("message")
    def _observe_message(self, change):
        self.show_temporary_message(change["new"])
```

Reply:
> Otherwise, why not follow my suggestion and use the class variable as a default value for the function argument?

What about my other suggestion?

Reply:
As the class variable still not have a chance to set it when creating the instance?

Reply:
@yakutovicha Could you look at the use of the `StatusHTML` widget here and let me know how to fix the problem that the message does not show again?

Reply:
Yep, this is done in c9214504b4e80a7a8b4d346bc5d695e08ea14520. Also, I made some changes in the `StatusHTML` widget itself, but those will be incorporated in the other PR.

Reply:
Not entirely. Please see other changes (including yours) here: https://github.com/aiidalab/aiidalab-widgets-base/pull/313

Reply:
addressed in #313